homeassistant/components/braviatv/media_player.py

```diff
@@ -58,6 +58,7 @@ class BraviaTVMediaPlayer(BraviaTVEntity, MediaPlayerEntity):
         | MediaPlayerEntityFeature.PLAY_MEDIA
         | MediaPlayerEntityFeature.BROWSE_MEDIA
     )
+    _attr_volume_step = 1 / 100
```

Comment:
Are we using 100 steps since it seems to be a percentage range in the client?

Reply:
Yes, that's the current behavior